net/dns/resolver/tsdns.go

```diff
@@ -62,6 +63,24 @@ type packet struct {
 	addr netaddr.IPPort // src for a request, dst for a response
 }
 
+// ResolverName is the configuration for a DNS resolver.
+type ResolverName struct {
+	// Addr is the address of the DNS resolver, one of:
+	//  - A plain IP address for a "classic" UDP+TCP DNS resolver
+	//  - An IP:port pair
```

Comment:
what does this mean? same as first bullet but alternate port? same port for both UDP+TCP?